.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please remove this folder.
Or please add .vscode in .gitignore

Reply:
ok !